src/probability/probability_mass_function/basic.lean

```diff
@@ -283,15 +298,12 @@ instance to_measure.is_probability_measure [measurable_space α] (p : pmf α) :
 variables [countable α] [measurable_space α] [measurable_singleton_class α]
   (p : pmf α) (μ : measure α) [is_probability_measure μ]
 
-lemma to_measure_to_pmf : p.to_measure.to_pmf = p :=
+@[simp] lemma to_measure_to_pmf : p.to_measure.to_pmf = p :=
 pmf.ext (λ x, by rw [← p.to_measure_apply_singleton x (measurable_set_singleton x),
   p.to_measure.to_pmf_apply])
 
 lemma to_measure_eq_iff_to_pmf_eq (μ : measure α) [hμ : is_probability_measure μ]:
   p.to_measure = μ ↔ μ.to_pmf = p :=
-begin
-  refine ⟨λ h, trans _ p.to_measure_to_pmf, λ h, trans (congr_arg _ h.symm) μ.to_pmf_to_measure⟩,
-  unfreezingI {induction h, refl}
-end
+by rw [← to_measure_inj, measure.to_pmf_to_measure, eq_comm]
```

Comment:
```suggestion
by rw [← to_measure_inj, measure.to_pmf_to_measure]
```
and change the statement accordingly. Swapping sides of the equality is undesirable.